hail_search/hail_search_query.py

```diff
@@ -59,6 +62,15 @@ class BaseHailTableQuery(object):
         XPOS: lambda r: [r.xpos],
     }
 
+    @classmethod
+    def load_globals(cls):
+        loaded_globals = {}
+        for genome_version in cls.GENOME_VERSIONS:
+            ht_path = cls._get_generic_table_path(genome_version, 'annotations.ht')
+            globals_ht = hl.read_table(ht_path).head(0).select()
+            loaded_globals[genome_version] = {k: hl.eval(globals_ht[k]) for k in cls.GLOBALS}
```

Comment:
Will `cls.GLOBALS` ever have more than a single element?  If so maybe faster to call `hl.eval()` only once (though it gets real jank):

```
dict(hl.eval(hl.Struct(**{k:globals_ht[x] for k in cls.GLOBALS})))
```

Reply:
it can, because we need the clinvar version which is in the version globals. I can make this one eval statement